packages/configs/default/index.json

```diff
@@ -3,8 +3,10 @@
   "transforms": {
     "types:*.{ts,tsx}": ["@parcel/transformer-typescript-types"],
     "*.{js,mjs,jsm,jsx,es6,ts,tsx}": [
+      "@parcel/transformer-react-refresh-babel",
```

Comment:
What’s the performance impact of including these extra transformers in the default config? If I’m building a non-React app, I don’t need these technically. If it’s minimal, then we should keep them here. Otherwise, we should investigate making it faster somehow or make a `@parcel/config-react` preset.

Reply:
Do you mean that the transformers themselves (the overhead) or the action that they perform?

Reply:
The overall cost